android/third_party_libs.md

```diff
@@ -10,13 +10,13 @@ Source: <https://github.com/square/retrofit>
 Overview of OkHttp: <http://square.github.io/okhttp> <br/>
 Source for OkHttp: <https://github.com/square/okhttp>
 
-### Otto
+### EventBus
 
-An event bus tailored to Android, Otto supports communication between distinct application components, when the use of a traditional interface is inadequate or cumbersome. 
+An event bus tailored to Android, greenrobot's EventBus supports communication between distinct application components; generally, the event bus should only be used when a traditional interface is inadequate or cumbersome, or an event is broadcast to multiple listeners at once.  
```

Comment:
right, I know that's your position, and I _generally_ agree with you; I don't oppose removing it. Are there any cases where you want strong decoupling, for which you'd entertain the idea of using the bus? The service-activity/fragment example is popular, but you can certainly connect to the service and treat it like an interface, though that connection step is async